src/posix.mak

```diff
@@ -440,8 +451,9 @@ toolchain-info:
 $G/glue.a: $(G_GLUE_OBJS) $(SRC_MAKE)
 	$(AR) rcs $@ $(G_GLUE_OBJS)
 
-$G/backend.a: $(G_OBJS) $(SRC_MAKE)
-	$(AR) rcs $@ $(G_OBJS)
+$G/backend.a: $(G_OBJS) $(G_DOBJS) $(SRC_MAKE)
+	$(HOST_DMD_RUN) -lib -of$G/backend.a $(G_OBJS)
+	$(HOST_DMD_RUN) -lib -of$G/backend.a $@ $(G_DOBJS)
```

Comment:
Now that the problems has been identified this could be reverted to `$(AR) rcs $@ $(G_OBJS)`?

Reply:
Why use `ar`? dmd can create libraries just fine. I put it on 2 lines as the log file was cutting off the end of the line.

Reply:
Found a good reason for using `$(AR)` - `gdmd` doesn't support it.
-> https://github.com/dlang/dmd/pull/8111